src/cljs/athens/views/app_toolbar.cljs

```diff
@@ -0,0 +1,228 @@
+    [:th]
+    [:th "Athens"]
+    [:th "Roam"]]
+   [:tbody
+    [:tr
+     [:td "Text Editing"]
+     [:td [feature-yes]]
+     [:td [feature-yes]]]
+    [:tr
+     [:td "Bidirectional Links"]
+     [:td [feature-yes]]
+     [:td [feature-yes]]]
+    [:tr
+     [:td "Timeline (Daily Notes)"]
+     [:td [feature-yes]]
+     [:td [feature-yes]]]
+    [:tr
+     [:td "Bookmarked Pages"]
+     [:td [feature-yes]]
+     [:td [feature-yes]]]
+    [:tr
+     [:td "Todos, Kanban, etc."]
+     [:td [feature-no]]
+     [:td [feature-yes]]]]])
+
+;(defn open-dialog! []
+;  (let [res (.showOpenDialogSync dialog (clj->js {:properties ["openFile"]
+;                                                  :filters [{:name "Transit" :extensions ["transit"]}]}))
+;        filepath (first res)]
+;    (when filepath
+;      (dispatch [:db/update-filepath filepath]))))
+;
+;(defn save-dialog! []
+;  (let [filepath (.showSaveDialogSync dialog (clj->js {:title "my-db"
+;                                                       :filters [{:name "Transit" :extensions ["transit"]}]}))]
+;    (dispatch [:db/update-filepath filepath])))
+
+(defn app-toolbar
+  []
+  (let [left-open? (subscribe [:left-sidebar/open])
+        right-open? (subscribe [:right-sidebar/open])
+        current-route (subscribe [:current-route])
+        ;db-filepath (subscribe [:db/filepath])
+        db-synced (subscribe [:db/synced])
+        import-modal-open? (r/atom false)
+        route-name (-> @current-route :data :name)]
+
+    (fn []
+      [:<>
+       [:header (use-style app-header-style)
+        [:div (use-style app-header-control-section-style)
+         [button {:active @left-open?
+                  :on-click #(dispatch [:left-sidebar/toggle])}
+          [:> mui-icons/Menu]]
+      ;; [separator] // for Electron implementation
+      ;; [button {:on-click-fn #(navigate :home)
+      ;;          :label [:> mui-icons/ChevronLeft]}]
+      ;; [button {:on-click-fn #(navigate :home)
+      ;;          :label [:> mui-icons/ChevronRight]}]
+         [separator]
+         [button {:on-click #(navigate :home)
+                  :active (when (= route-name :home) true)} [:> mui-icons/Today]]
+         [button {:on-click #(navigate :pages)
+                  :active (when (= route-name :pages) true)}
+          [:> mui-icons/FileCopy]]
+         [button {:on-click #(dispatch [:athena/toggle])
+                  :style {:width "14rem" :margin-left "1rem" :background (color :background-minus-1)}
+                  :active (when @(subscribe [:athena/open]) true)}
+          [:<> [:> mui-icons/Search] [:span "Find or Create a Page"]]]]
+
+        [:div (use-style app-header-secondary-controls-style)
+
+         ;(if @db-filepath
+         ;  [:h6 @db-filepath]
+         ;  [:h6 "Open a" [:code " .transit "] "DB or create a new one."])
+         ;[button {:on-click open-dialog!}
+         ; [:> mui-icons/FolderOpen]]
+         ;[button {:on-click save-dialog!}
+         ; [:> mui-icons/Save]]
```

Comment:
Delete. A dialog for these two options is shown if app cannot find file at filepath.